static/src/javascripts/projects/commercial/modules/prebid/slot-config.spec.js

```diff
@@ -29,6 +51,35 @@ describe('getSlots', () => {
 
     test('should return the correct slots at breakpoint M', () => {
         getBreakpointKey.mockReturnValue('M');
+        getCookie.mockReturnValue('UK');
```

Comment:
getCookie never returns "UK". Possible values are: "EU", "NA", "SA", "OC", "AS", "AF".

Reply:
thanks for pointing out, this test is testing behaviour of NON-US location (any other string than NA/na), but could be better to be more precise for better readability

Reply:
👍